app/vmselect/prometheus/export.qtpl

```diff
@@ -158,4 +158,17 @@
 	}
 	{% endif %}
 {% endfunc %}
+
+{% func convertValueToSpecialJSON(v float64) %}
+	{% if math.IsNaN(v) %}
+		null
```

Comment:
this line returns `null`, but is there a logic on parsing such values?

Reply:
made it compatible with import